python/pylibvw.cc

```diff
@@ -261,6 +105,35 @@ vw_ptr my_initialize_with_log(std::string args, py_log_wrapper_ptr py_log)
   return boost::shared_ptr<vw>(foo);
 }
 
+vw_ptr my_initialize_with_pyred(std::string args, py_log_wrapper_ptr py_log, py::object with_reduction)
```

Comment:
nit: `my`? who's?

Reply:
oh nevermind I see it's a convention in this file

Reply:
Sorry to revive this - I can't comment on the same line in a different thread.


Can the string be a const ref?